tools/requirements.txt

```diff
@@ -7,192 +7,191 @@
-paramiko==2.9.2; python_version >= "3.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version >= "3.7" or python_version >= "3.7"
+oauthlib==3.2.0; python_version >= "3.7"
+packaging==20.9; python_version >= "3.7"
+paramiko==2.9.2; python_version >= "3.7"
 parsedatetime==2.6; python_version >= "3.7"
 parso==0.8.3; python_version >= "3.7"
 pathlib2==2.3.7.post1; python_version >= "3.7"
 pexpect==4.8.0; python_version >= "3.7" and python_version < "4.0" or python_version >= "3.7" and sys_platform != "win32"
 pickleshare==0.7.5; python_version >= "3.7"
-pip==22.0.3; python_version >= "3.7"
+pip==22.0.4; python_version >= "3.7"
 pkginfo==1.8.2; python_version >= "3.7" and python_version < "4.0" or python_version >= "3.7"
-platformdirs==2.5.1; python_version >= "3.7" and python_full_version >= "3.6.2"
-pluggy==1.0.0; python_version >= "3.7" and python_full_version < "3.0.0" or python_version >= "3.7" and python_full_version >= "3.5.0" or python_version >= "3.7"
+pluggy==1.0.0; python_version >= "3.7"
 ply==3.11; python_version >= "3.7"
-poetry-core==1.1.0a6; python_version >= "3.7" and python_version < "4.0"
-poetry==1.2.0a2; python_version >= "3.6" and python_version < "4.0"
-prompt-toolkit==3.0.28; python_version >= "3.7" and python_full_version >= "3.6.2"
+poetry-core==1.1.0a7; python_version >= "3.7" and python_version < "4.0"
+poetry==1.2.0a2
+poetry==1.2.0a2; python_version >= "3.7" and python_version < "4.0"
+prompt-toolkit==3.0.28; python_version >= "3.7"
 protobuf==3.19.4; python_version >= "3.7"
 ptyprocess==0.7.0; python_version >= "3.7" and python_version < "4.0"
-py==1.11.0; python_version >= "3.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version >= "3.7"
-pyasn1-modules==0.2.8; python_version >= "3.7" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.7" or python_version >= "3.7"
-pyasn1==0.4.8; python_version >= "3.7" and python_version < "4" or python_version >= "3.7"
-pycparser==2.21; python_version >= "3.7" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.7"
+py==1.11.0; python_version >= "3.7"
+pyasn1-modules==0.2.8; python_version >= "3.7"
+pyasn1==0.4.8; python_version >= "3.7"
+pycparser==2.21; python_version >= "3.7"
 pygithub==1.55; python_version >= "3.7"
 pygments==2.11.2; python_version >= "3.7"
 pyjwt==2.3.0; python_version >= "3.7"
 pylev==1.4.0; python_version >= "3.7" and python_version < "4.0"
-pylint==2.12.2; python_version >= "3.7" and python_full_version >= "3.6.2"
-pynacl==1.5.0; python_version >= "3.7" or python_version >= "3.7"
+pylint==2.4.4; python_version >= "3.7"
+pynacl==1.5.0; python_version >= "3.7"
```

Comment:
The current CI failure is caused by `pylint` being downgraded here. Not sure why Poetry has decided to do this, looking into it.

Reply:
:+1: I see you opened #9229 to solve this.

Reply:
That's merged now, so hopefully `repin.sh` shouldn't crash anymore.

Try merge `master`, recreate the venv and re-run the pinning scripts. For me, doing this doesn't result in pylint being downgraded, so 🤞 .

Reply:
Nor for me :raised_hand_with_fingers_splayed: Thanks.